### DISCUSSION TITLE
number of temp sensors

### DISCUSSION THREAD

**Discussion by colly05** [KlipperScreen/KlipperScreen#982] [Ideas]
it seams to me that it is limited to 5 standard temp sensors that can show the colour next to the name after 5 the colour will not show. can we up the amount ?

**Comment by alfrix** [comment]
the colors are defined in hKlipperScreen/blob/master/styles/{theme}/style.conf
i selected colors for themes based on sensor type, but it's reasonable to assign a random color if it's not on the list

**Comment by colly05** [comment]
im not sure you know what i mean here is whats going on 
![IMG_1751](https://github.com/jordanruthe/KlipperScreen/assets/85044944/b77f746c-a61b-4b84-a072-b9027fce5a1e)

**Comment by alfrix** [comment]
i understood perfectly see https://github.com/jordanruthe/KlipperScreen/blob/master/styles/z-bolt/style.conf

i need to code a random color set to solve #944
